wagtail/admin/views/account.py

```diff
@@ -138,10 +138,18 @@ def render(self):
 
 class NameEmailSettingsPanel(BaseSettingsPanel):
     name = "name_email"
-    title = gettext_lazy("Name and Email")
+    # title = gettext_lazy("Name and Email")
```

Comment:
Best to remove this line completely - no need to leave a commented out line in like this.

Reply:
Just a reminder to remove this line completely.